src/pages/home-page/HomePage.js

```diff
@@ -5,16 +5,21 @@ import "./home-page.scss";
 
 const HomePage = () => (
   <div className="home-page">
-    <div className="login">
-      <img src={logo} alt="Logo" className="logo" />
+    <section className="login">
+      <figure>
+        <img src={logo} alt="Logo" className="logo" />
+      </figure>
       <LoginForm />
-    </div>
-    <div className="signup">
+      <nav className="forgot-password-link">
```

Comment:
I'm not sure if you have to wrap every link with a nav tag. It's suppose be used only for the main navigation on your site, like a menu or header navbar. I'm not saying is wrong though, maybe it's not the best recommended practice. Good job anyway!

Reply:
Do you mean to change the nav by a div?

Reply:
In this case, I don't this you need the nav. If you need a wrapper for styling purposes only, you could use a div